lib/Controller/WopiController.php

```diff
@@ -485,7 +485,13 @@ public function putFile($fileId,
 
 			$content = fopen('php://input', 'rb');
 
+			$freespace = $file->getParent()->getFreeSpace();
+			$contentLength = $this->request->getHeader('CONTENT_LENGTH');
```

Comment:
```suggestion
			$contentLength = (int)$this->request->getHeader('Content-Length');
```

Reply:
The naming is mostly to match the actual http header, uppercase and underscore are then converted in our getHeader method in the server to match the PHP pattern